src/hmem.c

```diff
@@ -48,6 +48,11 @@ struct ofi_hmem_ops {
 
 static pthread_mutex_t init_lock = PTHREAD_MUTEX_INITIALIZER;
 
+const char *hmem_iface_to_str[] = {
+	[FI_HMEM_SYSTEM] = "system",
+	[FI_HMEM_CUDA] = "cuda",
+};
```

Comment:
I think we'll want this accessible from the fi_tostr() function.

Reply:
Will do.